lua/lsp/providers/typescriptreact.lua

```diff
@@ -0,0 +1,11 @@
+local opts = {
```

Comment:
lvim.lang.javascript and lvim.lang.typescript are available through autocomplete but their react analogues are not: lvim.lang.javascriptreact and lvim.lang.typescriptreact.  And any attempt to use typescriptreact results in a config error.

Reply:
Oh I forgot to document this, I added a new variable temporarily, to set up the common languages quickly: https://github.com/LunarVim/LunarVim/blob/71aa17810261b0a41130de19df692c3ad675b75f/lua/config/defaults.lua#L34

I'm not yet settled if we should keep it or not. 

Can you append the values you want to it and if it solves the issue?